posthog/temporal/data_imports/sources/common/graphql_source/typing.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any
+
+GraphQLResponse = dict[str, Any]
+
+
+class GraphQLResource:
```

Comment:
nit: I'd prefer for us to not use the term `Resource` - but instead use `Source`. Resource is a legacy term from when we were using DLT